needle/modules/dynamic/watch/traffic_capture.py

```diff
@@ -0,0 +1,130 @@
+import paramiko
+import socket
+import select
+
+from core.framework.module import BackgroundModule
+from multiprocessing import Process
+
+
+class Module(BackgroundModule):
+    meta = {
+        'name': 'Title',
+        'author': '@Andrea Amendola',
+        'description': 'Description',
+        'options': (
+            ('port', '8080', True,
+             'port of the service that will handle the captured traffic'),
+        ),
+    }
+
+    # ==================================================================================================================
+    # UTILS
+    # ==================================================================================================================
+    def module_pre(self):
+        return BackgroundModule.module_pre(self, bypass_app=True)
+
+
+    # ==================================================================================================================
+    # REMOTE PORT FORWARDING
+    # ==================================================================================================================
+
+    def _handler(self, chan, host, port):
```

Comment:
I agree with @HenryHoggard, there should be a cleaner (more abstract) way to implement this